frigate/record/cleanup.py

```diff
@@ -140,6 +148,14 @@ def expire_recordings(self) -> None:
                     Path(recording.path).unlink(missing_ok=True)
                     deleted_recordings.add(recording.id)
 
+                    # delete timeline entries relevant to this recording segment
+                    for time in [
```

Comment:
Can't you do this in a single delete statement rather than a loop?

where:
- `timestamp between recording start/end`
- `timestamp < expire_date`
- `camera == camera`

Reply:
Yeah, I was hoping this would reduce db activity for timeline but after building it out it seems that would not have much of an impact and would be a lot cleaner